apps/meteor/client/omnichannel/tags/TagsPage.tsx

```diff
@@ -12,6 +13,9 @@ const TagsPage = () => {
 	const router = useRouter();
 	const context = useRouteParameter('context');
 	const id = useRouteParameter('id');
+	const handleCloseContextualbar = useEffectEvent(() => {
```

Comment:
```suggestion

```

Reply:
This ones fine, function re-used at other places.